### DISCUSSION TITLE
Calculate Directory Size

### DISCUSSION THREAD

**Discussion by Razboy20** [pterodactyl/panel#3066] [Feature Requests]
This could be implemented by adding an option to the dropdown menu which could be, for example, Calculate Directory Size, which would populate the size column of that directory like regular files have.

Seems like it should not be too difficult, although I am not sure.

**Comment by DaneEveritt** [comment]
This explodes fantastically when you inevitably have a directory with millions of tiny files in it, the crush that occurs just to calculate that is unnecessary in almost all cases.
